reference/constraints/Callback.rst

```diff
@@ -4,8 +4,8 @@ Callback
 The purpose of the Callback constraint is to create completely custom
 validation rules and to assign any validation errors to specific fields
 on your object. If you're using validation with forms, this means that you
-can make these custom errors display next to a specific field, instead of
-simply at the top of your form.
+can make these custom errors display next to a specific field, instead at
+the top of your form.
```

Comment:
According to Grammarly `[...] instead of at the` is correct, otherwise you change the subject `[...] instead, at a top location` (which is something completely different). Maybe `[...] instead of, at the`? 

@weaverryan can you help here :)

Reply:
Thank you for the detailed explanation, could you please write down the complete sentence how you would rewrite it?

Reply:
my suggestion: 

If you're using validation with forms, this means that instead of displaying custom errors at the top of the form, you can display them next to the field they apply to.